challenges/BackEnd/README.md

```diff
@@ -0,0 +1,46 @@
+# Headstorm Backend Challenge
```

Comment:
Hi, thanks for submitting your code for this challenge. 

It is difficult to navigate through 1894 files to read and understand code. Can you update your code to include a .gitignore?

Reply:
Yes, I have made the appropriate changes. I had originally included a clean virtual environment of python for the Backend section, which has now been removed. The main files to consider are:
BackEnd\app\challenge.py
FrontEnd\pages\contact_us.html
FrontEnd\pages\submit_form.php
FrontEnd\pages\validate_captcha.php
Database\src\main\java\Javier\DatabaseMigrationTool\DatabaseMigration.java
Database\src\main\java\Javier\DatabaseMigrationTool\NoSQLInfoObject.java